controllers/BookmarksController.js

```diff
@@ -0,0 +1,121 @@
+
+import Models from '../models';
+import StatusResponse from '../helpers/StatusResponse';
+
+const { bookmarks, articles } = Models;
+/** @description usersController class
+ * @return {object} the response object
+ * @public
+ */
+class BookmarksController {
+  /** @description create a bookmark
+   * @param {string} req is the request parameter
+   * @param {string} res is the response parameter
+   * @return {object} the response object
+   * @public
+   */
+  static async create(req, res) {
+    const { userId } = res.locals.user;
+    const { articleId } = req.params;
+    let { title } = req.body;
+
+    if (!title) {
+      title = `untitled ${articleId}`;
```

Comment:
The team agreed to use the article title as the default bookmark title. Why use `untitled $articleId`?

Reply:
done